cli/bpmetadata/proto/bpmetadata_ui.proto

```diff
@@ -94,6 +94,20 @@ message DisplayVariable {
   // Optional. UTF-8 text. No markup. At most 128 characters.
   // Gen: manually-authored
   string validation = 15; // @gotags: json:"validation,omitempty" yaml:"validation,omitempty"
+
+  // Property subtext, displayed below the title.
+  // Gen: manually-authored
+  string subtext = 16; // @gotags: json:"subtext,omitempty" yaml:"subtext,omitempty"
+
+  // Labels for enum values.
```

Comment:
Where are these labels used?

Reply:
Defined in the current view model (config.proto) [here](http://shortn/_O5NLV4VVZE)

Used currently for defining enum values for particular properties. 

Example in a DM product schema [here](http://shortn/_F7LXUdPmDJ)

Reply:
We want to add these here so that they are available for the properties that should have enum values present, but are not currently available yet for terraform setup due to this field not being defined. (we are currently [ignoring](http://shortn/_SKVsQ6UPiH) during conversion)

Reply:
Thanks for the explanation.